src/borg/archiver/compact_cmd.py

```diff
@@ -176,32 +176,33 @@ def build_parser_compact(self, subparsers, common_parser, mid_common_parser):
             Free repository space by deleting unused chunks.
 
             borg compact analyzes all existing archives to find out which repository
-            objects are actually used (referenced). It then removes all unused objects
-            to free repository space.
+            objects are actually used (referenced). It then deletes all unused objects
+            from the filesystem to free space.
```

Comment:
no "from the filesystem". borg supports all kinds of storages now (including cloud storage) and we can't know whether they use a filesystem.

Reply:
Hmmm... :thinking: Being pedantic, that's indeed also true for any cloud storage, it's just a matter of how many other layers are in between :laughing: Anyway, you're right, it's bad wording. I don't like "free repository space" very much though, changed it to "from the repository to free space".